### PR TITLE
build: Bump Alpine version

### PR DESCRIPTION
**Pull request by arkadyan**
Also run `apk upgrade`.

No ticket.

### CODE REVIEW

**Comment by arkadyan** [comment]
[Successful dev deploy](https://github.com/mbta/document_viewer/runs/6261874713?check_suite_focus=true)

File: Dockerfile [hunk 1]
```diff
@@ -36,11 +36,13 @@ COPY --from=assets-builder /root/priv/static ./priv/static
 RUN mix do compile --force, phx.digest, release
 
 # Finally, use an Alpine container for the runtime environment
-FROM alpine:3.13.1
+FROM alpine:3.15.4
 
 RUN apk add --update libssl1.1 ncurses-libs bash curl dumb-init \
   && rm -rf /var/cache/apk
 
+RUN apk upgrade --no-cache --update
```
[the remark this answers is not in the document]
Comment (arkadyan) [reply Dockerfile]: Yep, good call

**Review by paulswartz — APPROVED** [review]
🍰 once the last update is made and you test that the new Dockerfile also deploys fine.

File: Dockerfile [hunk 2]
```diff
@@ -36,9 +36,10 @@ COPY --from=assets-builder /root/priv/static ./priv/static
 RUN mix do compile --force, phx.digest, release
 
 # Finally, use an Alpine container for the runtime environment
-FROM alpine:3.13.1
+FROM alpine:3.15.4
```
[the remark this answers is not in the document]
Comment (arkadyan) [reply Dockerfile:39]: Oops, good catch, thanks.

**Pull request merged**